ci/runtests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Please also make this an executable script, and change this shebang to be `#!/bin/bash -ex`